### DISCUSSION TITLE
Is it possible to use custom Bgd peaks when calculating DeviationsMatrix?

### DISCUSSION THREAD

**Discussion by devbiosnoww** [GreenleafLab/ArchR#2001] [Questions / Documentation]
Hi ArchR team,

Thank you so much for developing and maintaining such a wonderful package!
I'm wondering when adding background peaks for calculating DeviationsMatrix, besides the current "chromVAR or ArchR" option, can I also use a set of custom (user-supplied) peaks? 

Thanks a lot,
Snow

**Comment by rcorces** [comment]
Please see the function documentation
https://www.archrproject.com/reference/addDeviationsMatrix.html
